shuriken/bindings/Python/setup.py

```diff
@@ -56,6 +56,27 @@ def build_libraries():
     else:
         logger.warning("CMake build is only supported on Linux for now.")
 
+    if platform.system() == 'Darwin':
+        try:
+            with change_directory(BUILD_FOLDER):
```

Comment:
i think if the code is identical we should move them to the same if condition.

Also here we're letting the logger leak out a warning of linux being supported even if we're compiling on Darwin

Reply:
Morning I just pushed the change, we will need to keep an eye on the setup.py script as MacOS may change <sigh> abruptly

Reply:
Yes i agree, good work! :)